packages/flutter/lib/src/widgets/scroll_position.dart

```diff
@@ -562,6 +562,26 @@ abstract class ScrollPosition extends ViewportOffset with ScrollMetrics {
   @override
   void jumpTo(double value);
 
+  @override
+  Future<void> moveTo(double to, {
```

Comment:
We should probably provide a new doc comment for this method, otherwise dartdocs will show the dartdoc of the superclass, which would incorrectly claim that `clamp` is ignored?

Reply:
The super class does say
>  .. but subclasses like [ScrollPosition] handle it by adjusting [to] to prevent over or under scrolling

But you're right, it would clearer if there was new doc here.